lte/gateway/python/magma/mobilityd/subscriberdb_client.py

```diff
@@ -20,30 +20,41 @@
 import logging
 
 
+class NetworkInfo:
+    def __init__(self, gw_ip: Optional[str] = None, gw_mac: Optional[str] = None,
+                 vlan: int = 0):
+        gw_ip_parsed = None
+        try:
+            gw_ip_parsed = ipaddress.ip_address(gw_ip)
+        except ValueError:
+            logging.debug("invalid internet gw ip: %s", gw_ip)
+
+        self.gw_ip = gw_ip_parsed
```

Comment:
what if `gw_ip_parsed` can't be parsed and None is assigned? do we do additional checks for this?

Reply:
That is handled. in many cases the gw would be None.